litmus-portal/frontend/src/views/Home/TotalWorkflows/index.tsx

```diff
@@ -51,16 +50,18 @@ const TotalWorkflows: React.FC<TotalWorkflowProps> = ({
               {max}
             </Typography>
           </div>
-          <AnalyticsLinearProgressBar
+          {/* {console.log(average)} */}
+          <LinearProgress variant="determinate" value={average} />
+          {/* <AnalyticsLinearProgressBar
             value={average}
             maxValue={max}
```

Comment:
Can you remove this commented code, since our codebase has already lot of code like this.

Reply:
Removed it

Reply:
Resolved :+1: